Echo.Process.Owin/Echo.Process.Owin.csproj

```diff
@@ -3,8 +3,8 @@
     <NoWarn>1701;1702;1705;IDE1006</NoWarn>
   </PropertyGroup>
   <PropertyGroup>
-    <TargetFramework>net48</TargetFramework>
-    <PackageVersion>2.1.25-beta</PackageVersion>
+    <TargetFramework>net461</TargetFramework>
+    <PackageVersion>2.1.7-beta</PackageVersion>
```

Comment:
Don't change the package-version, my tooling adds 1 to the last version when I do a deployment, so leave this as-is

Reply:
Fixed.